### PR TITLE
Add ut for querycoord's meta

### PR DESCRIPTION
**Pull request by xige-16**
fix issue: #7192
Signed-off-by: xige-16 <xi.ge@zilliz.com>

### CODE REVIEW

File: internal/querycoord/meta.go [hunk 1]
```diff
@@ -599,6 +599,11 @@ func (m *MetaReplica) GetQueryChannel(collectionID UniqueID) (string, string) {
 		QueryChannelID:       allocatedQueryChannel,
 		QueryResultChannelID: allocatedQueryResultChannel,
 	}
+	err := saveQueryChannelInfo(collectionID, queryChannelInfo, m.client)
+	if err != nil {
+		//TODO:: deal err info
+		log.Error("GetQueryChannel: save channel to etcd error", zap.Error(err))
```
Comment (czs007) [inline internal/querycoord/meta.go:605]: not return?

Comment (xige-16) [reply internal/querycoord/meta.go:605]: TODO in pr #7679, ut is hard to add if return err in this pr?

Comment (xige-16) [reply internal/querycoord/meta.go:605]: There is very little diff in this pr. If err is returned, ut needs to be added to cover the case of etcd write failure.

**Pull request merged**